openedx/features/edly/admin.py

```diff
@@ -0,0 +1,22 @@
+"""
+Django Admin pages for Edly.
+"""
+
+from django.contrib import admin
+
+from .models import EdlyOrganization, EdlySubOrganization
+
+
+class OrganizationAdmin(admin.ModelAdmin):
```

Comment:
It would be better if we keep model name and admin class name consistent.

Reply:
Feedback Implemented